### ISSUE TITLE
Cursor focus still in Chrome's address bar after pressing ENTER

### ISSUE THREAD

**Issue by clarkwang** [ChrisNZL/Fauxbar#66]
I need to explicitly click on the opened page so I can use <kbd>Up/Down</kbd> keys for scrolling. Could we make the address bar lose focus after pressing ENTER?

**Comment by ChrisNZL** [comment]
_tl;dr: Press <kbd>Tab</kbd> afterwards to shift focus off of Chrome's Omnibox and onto the page._

---

Which version of Chrome and operating system are you using?

Is this issue when using <kbd>F</kbd>+<kbd>Spacebar</kbd> from Chrome's Omnibox (which is really clunky as of Chrome 97)?

Or is this from using Fauxbar's Address Box?

Could you please screenshot, or record a GIF (I recommend [ShareX](https://getsharex.com/)), so I can see what you mean (just to make sure I'm understanding you correctly)?

If this is from Chrome's Omnibox, Chrome's Omnibox + extension search behaviour has gotten worse over time with newer Chrome versions. Chrome keeps making it more cumbersome to initiate a search. If the caret is in the Omnibox, Fauxbar can't automatically change focus to the page, and yes, it's annoying how the Omnibox retains focus despite the search executing and the page loading.

---

In Chrome 97, typing `f stuff` merely _suggests_ for me to search using Fauxbar:

![image](https://user-images.githubusercontent.com/280814/150016802-e495b555-7f00-4b24-a6da-34884af56622.png)

whereas in previous versions of Chrome, typing `f stuff` would immediately jump to the Omnibox+Fauxbar search and have `stuff` be my search query.

(however, the focus would still remain inside the Omnibox; this is out of Fauxbar's control)

But now, with Chrome 97, it seems the user has to **click the <kbd>🔍 Fauxbar</kbd> button** to initiate the Omnibox+Fauxbar search; pressing <kbd>Down Arrow</kbd> then <kbd>Enter</kbd> searches with Google, rather than Fauxbar. This is poor design by the Chromium team.

---

If I **click** the suggested <kbd>🔍 Fauxbar</kbd> entry, this initiates the Omnibox+Fauxbar search:

![image](https://user-images.githubusercontent.com/280814/150017622-59642041-608f-4244-bb23-6cfe1f128d9f.png)

But afterwards, Chrome's Omnibox retains focus:

![image](https://user-images.githubusercontent.com/280814/150017766-b72f58b6-d80b-4869-8391-af31a8c99cc9.png)

... and there is nothing Fauxbar can do to change focus.

This is kind of why I built Fauxbar; a whole page with control over how Address Box results can be interacted with, rather than using Chrome's limited Omnibox.

The whole <kbd>F</kbd>+<kbd>Spacebar</kbd> shortcut was added as a convenience feature (just because Chrome offers it in their [chrome.omnibox API](https://developer.chrome.com/docs/extensions/reference/omnibox/)), but in recent versions of Chrome over the years, the Omnibox has gotten less and less convenient.

---

If you press <kbd>Tab</kbd> afterwards, you can at least focus the page without clicking.

**Comment by clarkwang** [comment]
> Is this issue when using F+Spacebar from Chrome's Omnibox (which is really clunky as of Chrome 97)?

Right it's from Chrome's address bar. I'm using Fauxbar Lite.

> This is kind of why I built Fauxbar; a whole page with control over how Address Box results can be interacted with, rather than using Chrome's limited Omnibox.
> 
> The whole F+Spacebar shortcut was added as a convenience feature (just because Chrome offers it in their [chrome.omnibox API](https://developer.chrome.com/docs/extensions/reference/omnibox/)), but in recent versions of Chrome over the years, the Omnibox has gotten less and less convenient.

OK I'll switch to Fauxbar and configure it as the default New Tab page.

Thanks.

**Comment by clarkwang** [comment]
Have been trying for some time. Finally I'd still use Fauxbar Lite but I configured a keyboard shortcut (I'm using `Alt-T`) to *Open Fauxbar Lite's Address Box in a new tab*. I also added a new Chrome keyword `fx` to open Fauxbar Lite in current Tab. So 4 use cases in total:

1. Press `Ctrl-T` to open Chrome's new Tab page.
1. Press `Alt-T` to open Fauxbar in a new Tab.
2. Go to Chrome's address bar and enter `f<SPACE>` to use Fauxbar in Chrome's address bar.
1. Go to Chrome's address bar and enter `fx<ENTER>` to open Fauxbar in current Tab.

**Comment by clarkwang** [comment]
> But now, with Chrome 97, it seems the user has to **click the 🔍 Fauxbar button** to initiate the Omnibox+Fauxbar search; pressing Down Arrow then Enter searches with Google, rather than Fauxbar.

Fauxbar Lite works fine for me with Chrome 97. It'll automatically become this after I input `f<SPACE>`:

![image](https://user-images.githubusercontent.com/157076/150112255-8b890d38-4ba5-45e3-a7b4-09a55a81ebd9.png)
